locale/menu.xml

```diff
@@ -0,0 +1,750 @@
+<?xml version="1.0"?>
+<menu xmlns="http://ledgersmb.org/xml-schemas/menu" id="0" label="Top-level">
```

Comment:
When I tried to load https://ledgersmb.org/xml-schemas/menu in a browser it returned a page not found.  how can I view this?

Reply:
The URL is nothing more than a namespace. The same happens in the 'initial-data.xml' and CoA xml files. I have the XSD added to my tree locally. I'll push that when the tests complete so I know the tests work.

Reply:
Pushed now.